### PR TITLE
copy repository for partner's review 

### PR DESCRIPTION
**Pull request by supreeti**
Copy code from ToDoList project for Coding partner review.

- [ ] Follow Javascript best practice
- [ ] Follow DRY,KISS, YAGNI rules
- [ ] Follow HTML & CSS best practice
- [ ] Use Webpack to bundle code

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,126 @@
...
+- Html
+- CSS
+- Javascript ES6
+- Webpack
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://github.com/supreeti/ToDoList-PartnerReview/)
```
Comment (ragangithub) [inline README.md]: The live demo link isn't working properly. The link must direct us to the deployed version, not to GitHub

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  e.preventDefault();
+  const todos = getList();
+  const toAdd = document.getElementById('addTasks').value;
+  const todoTask = {
+    index: todos.length,
+    description: toAdd,
+    completed: false,
+  };
+
+  if (toAdd !== '') {
```
Comment (ragangithub) [inline src/index.js]: It is best practice to use conditional short hands. Use this approach only for Boolean values and if you are sure that the value will not be undefined or null. for this case, you can change it to 'if(todo)'.

File: src/modules/DeleteList.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const deleteList = (index) => {
+  const toDoList = document.getElementById(`todo${index}`);
+  if (toDoList !== null) {
```
Comment (ragangithub) [inline src/modules/DeleteList.js]: It is best practice to use conditional short hands. Use this approach only for Boolean values and if you are sure that the value will not be undefined or null. for this case, you can change it to "if(todoList)'".

**Review by ragangithub — CHANGES_REQUESTED** [review]
## Good Highlights✔️
- No linter error
- correct PR description
- Project runs well
- Adding, editing and removing of tasks is implemented
- Changes are saved to the local storage
# Changes required ♻️ 
please make changes to suggested comments

File: src/index.js (hunk 2 above)
Comment (supreeti) [reply src/index.js]: Thanks for suggesting the changes..Resolved it.

File: README.md (hunk 1 above)
Comment (supreeti) [reply README.md]: Updated the correct link.

File: src/modules/DeleteList.js (hunk 3 above)
Comment (supreeti) [reply src/modules/DeleteList.js]: Resolved code.

**Review by ragangithub — APPROVED** [review]
- Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

# Highlights
Linters are passing 👍
Follows JavaScript best Practices 
The project is running successfully 👍
PR is well descriptive 👍

**Comment by supreeti** [comment]
Thanks🙏 😀

**Pull request merged**